### ISSUE TITLE
YBAF, update VFR approach point 'PKR'

### ISSUE THREAD

**Issue by steve-gehle** [vatpac-technology/sops#412]
Update references in [YBAF SOP](https://sops.vatpac.org/aerodromes/classd/Archerfield/#vfr-arrival-procedures) from 'Logan Bridge' to 'Logan Motorway' for VFR approach point 'PKR'

https://www.airservicesaustralia.com/aip/current/ersa/FAC_YBAF_28NOV2024.pdf

I'm happy to contribute this change via PR, please let me know if OK to proceed.

![image](https://github.com/user-attachments/assets/2506eba8-768c-47ab-94e6-35fdb356bf1c)

![image](https://github.com/user-attachments/assets/7d873586-065b-4a45-9fc0-eaca9920f1c6)

**Comment by mattkelly4** [comment]
@steve-gehle yep that sounds great. I don't have any experience with YBAF but can't seem to find a Logan Bridge, so I assume they are referring to the motorway. Have you flown in here IRL?

**Comment by steve-gehle** [comment]
> @steve-gehle yep that sounds great. I don't have any experience with YBAF but can't seem to find a Logan Bridge, so I assume they are referring to the motorway. Have you flown in here IRL?

I haven't flown in from this VFR approach point (only TAR) but am familiar with the local area. Crossing the 'Logan Motorway' as a visual waypoint is accurate, as 'Logan Bridge' is not.

I'll get a PR going.

**Comment by mattkelly4** [comment]
Hey @steve-gehle , absolutely no rush from our end, just curious how you ended up going with this? Do you want us to put a fix in place or are you still happy to do yourself?

**Comment by steve-gehle** [comment]
Thanks for the bump, I’ll get to this soon

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
